src/devices/funkbus.c

```diff
@@ -0,0 +1,197 @@
+/** @file funkbus.c
+ *  @author Markus Sattler
+ *  @date 2021-12-04
+ *
+ *  @brief Funkbus / Instafunk
+ *  used by Berker, Jira, Jung and may more
+ *  developed by Insta GmbH
+ *
+ *  @copyright Copyright (C) 2021 Markus Sattler
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ */
+
+/*
```

Comment:
this should be a doc comment with valid markdown, see other files

Reply:
changed